### ISSUE TITLE
A pop-up checklist (using javascript)

### ISSUE THREAD

**Issue by brianek** [codefordc/expungement-dc#188]
What: a pop-up checklist that prompts the user (new attorney) 
Why: make sure they have avoided the most common areas to assessing the eligibility of a record to seal. 
How: it should be like a pop-up checklist that requires the user to check off one by one that they have not made that common error.

**Comment by jmezetin** [comment]
Do you have a list of items that will need to be on the checklist?

**Comment by brianek** [comment]
Yes. Here are some examples. The checklist should be easy to edit from github however (because the law may change). Also, potentially, if possible, the common mistakes should come up based on how they answered the questions. For example, if they indicated that the person was charged with an assault of someone related by blood, there is no need to tell them that they should have considered that. 

A list of common mistakes in sealing:
1) Did you inquire as to whether or not a person has an assault charge for harming someone dating their ex-girlfriend or boyfriend? Assaulting someone dating your ex-girlfriend or boyfriend is a charge, either a felony or misdemeanor, that is not eligible for sealing under the current law. 

2) Did you check with the client as to whether they were required to make a contribution to the victims fund and successfully made said contribution? If an individual was required to make a contribution & did not do so, then they cannot be considered "off-papers" at all. To start the clock on "off papers" they must make a contribution to the victim's fund. 

3) For a client trying to seal a non-conviction, did you make sure to check if they had any other convictions?  If they have convictions, 5 years or 10 year, for a misdemeanor and felony, respectively)  had to have passed for them to be eligible to seal that non-conviction.

**Comment by jmezetin** [comment]
How does this look? http://jmezetin.github.io/clean-slate/organizations/login.html#/Home 

I'm not sure where in the application it should display.
